pcli/src/main.rs

```diff
@@ -101,13 +107,20 @@ async fn main() -> Result<()> {
     }
 
     match opt.cmd {
-        Command::Tx { key, value } => {
+        Command::Tx(Tx::Send) => {
             let spend_key = load_wallet(&wallet_path);
-            let _local_storage = state::ClientState::new(spend_key);
+            let mut local_storage = state::ClientState::new(spend_key);
+
+            // TODO: Set fee on CLI (part of issue #138)
+            let fee = 0;
+            let dummy_tx = local_storage.new_transaction(&mut OsRng, fee)?;
+            let serialized_tx: Vec<u8> = dummy_tx.into();
 
             let rsp = reqwest::get(format!(
-                r#"http://{}:{}/broadcast_tx_async?tx="{}={}""#,
-                opt.node, opt.abci_port, key, value
+                r#"http://{}:{}/broadcast_tx_async?tx="{}""#,
```

Comment:
Per https://docs.tendermint.com/master/tendermint-core/using-tendermint.html#formatting , it looks like specifying the transaction with an `0x` prefix will cause Tendermint to interpret it as hex-encoded binary data, and pass it to the application as binary.